### PR TITLE
RQ045 scripts

### PR DESCRIPTION
**Pull request by maiparkes58**
Just some work I did with RQ045 study. Can we save in master to use later if needed?

### CODE REVIEW

**Comment by rw251** [comment]
I'm happy to do that, but is it not just as easy to keep this branch and return to it when and if needed?

**Review by Georgetilston — APPROVED** [review]

File: projects/045 - Chen/template-sql/patients.template.sql [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+IF OBJECT_ID('tempdb..#GPEvents') IS NOT NULL DROP TABLE #GPEvents;
+SELECT gp.FK_Patient_Link_ID, EventDate, SuppliedCode, FK_Reference_Coding_ID, FK_Reference_SnomedCT_ID, [Value]
+INTO #GPEvents
+FROM SharedCare.GP_Events gp
+INNER JOIN #AllCodes a on a.Code = gp.SuppliedCode
+WHERE gp.FK_Patient_Link_ID IN (SELECT FK_Patient_Link_ID FROM #Patients)
+and EventDate BETWEEN @MinDate and @IndexDate
+-- 16 mins to run for 1000 patients
+
+-- add index
```
Comment (rw251) [inline projects/045 - Chen/template-sql/patients.template.sql:55]: Either delete this line, or add an index.

File: projects/045 - Chen/template-sql/patients.template.sql [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+INTO #SevereMental
+FROM #GPEvents
+WHERE (
+  FK_Reference_Coding_ID IN (SELECT FK_Reference_Coding_ID FROM #VersionedCodeSets WHERE Concept = 'severe-mental-illness' AND Version = 1) OR
+  FK_Reference_SnomedCT_ID IN (SELECT FK_Reference_SnomedCT_ID FROM #VersionedSnomedSets WHERE Concept = 'severe-mental-illness' AND Version = 1)
+);
+
+
+-- Select ethnicity and death date from PatientLink table================================================================================================================================
+IF OBJECT_ID('tempdb..#PatientLinkTable') IS NOT NULL DROP TABLE #PatientLinkTable;
```
Comment (rw251) [inline projects/045 - Chen/template-sql/patients.template.sql:110]: This is not used and can be deleted. the #Patients table already has this info

File: projects/045 - Chen/about.md
Comment (rw251) [inline projects/045 - Chen/about.md:1]: Can you update this.

File: shared/clinical-code-sets/procedures/mmr-vaccination/1/README.md
Comment (rw251) [inline shared/clinical-code-sets/procedures/mmr-vaccination/1/README.md:1]: I can't see MMR anywhere in the extract scripts. Fine to merge it here, but just checking it shouldn't be in the template sql.

**Review by rw251 — APPROVED** [review]
A few changes to make, but approved otherwise.

**Pull request closed without merging**